aiida/daemon/tasks.py

```diff
@@ -139,5 +139,5 @@ def manual_tick_all():
     submit_jobs()
     update_jobs()
     retrieve_jobs()
-    execute_steps() # legacy workflows
+    workflow_stepper() # legacy workflows
```

Comment:
I see, and I think I understand the problem. It's because in this new way we're calling the celery task.
Let's put it back to what it was for now. (better would be to have the task just call a function, and here call that function, with all the logic for setting the timestamps - both here and for the other functions.

Reply:
Yes, this was the conclusion of the discussion that I had with Sebastiaan a few mins ago.